{{cookiecutter.project_slug}}/.gitignore

```diff
@@ -2,6 +2,8 @@
 target/
 dbt_modules/
 .config/.user.yml
+# Remove if you have no secrets in the file in ./config/
```

Comment:
No need for "/**/". In any case, we should not ignore this file as this will break the CI pipeline.

Reply:
I saw this as a safety thing and mention this in the profile.yml
Not everyone will use CI, especially a new user and IMO it is likely a user will update that file and commit it to git.
If they are advanced enough to use CI, they will know what they need to do to .gitignore.

Reply:
Yeah, It would be safer if we just avoid putting credentials in .config/profiles.yml. I don't see why breaking CI if it could work out of the box.

Reply:
@ssassi I now have 2 profiles.yml and I move / rename the user one to root and ignore that for git